AllReadyApp/Web-App/AllReady/Areas/Admin/Controllers/SkillAdminController.cs

```diff
@@ -0,0 +1,126 @@
+﻿using AllReady.Models;
+using AllReady.Security;
+using Microsoft.AspNet.Authorization;
+using Microsoft.AspNet.Mvc;
+using System.Linq;
+
+namespace AllReady.Areas.Admin.Controllers
+{
+    [Area("Admin")]
+    [Authorize("SiteAdmin")]
+    public class SkillController : Controller
+    {
+        private readonly IAllReadyDataAccess _dataAccess;
+
+        public SkillController(IAllReadyDataAccess dataAccess)
+        {
+            _dataAccess = dataAccess;
+        }
+
+        private ViewResult WithSkills(ViewResult view)
+        {
+            view.ViewData["Skills"] = _dataAccess.Skills;
+            return view;
+        }
+
+        public IActionResult Index()
+        {
+            return View(_dataAccess.Skills);
+        }
+
+        public IActionResult Create()
+        {
+            return WithSkills(View("Edit"));
+        }
+
+        [HttpPost]
+        [ValidateAntiForgeryToken]
+        public IActionResult Create(Skill skill)
+        {
+            if (!User.IsUserType(UserType.SiteAdmin))
+            {
+                return new HttpUnauthorizedResult();
+            }
```

Comment:
This is not necessary because you have the Authorize("SiteAdmin") attribute on the controller.  You can remove this form each action method